hnn_core/network.py

```diff
@@ -617,6 +623,29 @@ def _attach_drive(self, name, drive, weights_ampa, weights_nmda, location,
         # Every time pos_dict is updated, gid_ranges must be updated too
         self._update_gid_ranges()
 
+        # Update connectivity_list
+        nc_dict = {
+            'A_delay': 1.,
+            'threshold': self.params['threshold'],
+        }
+        receptors = ['ampa', 'nmda']
+        if drive['type'] == 'gaussian':
+            receptors = ['ampa']
+        # conn-parameters are for each target cell type
+        for target_cell, drive_conn in drive['conn'].items():
+            for receptor in receptors:
+                if len(drive_conn[receptor]) > 0:
+                    nc_dict['lamtha'] = drive_conn[
+                        receptor]['lamtha']
+                    nc_dict['A_delay'] = drive_conn[
+                        receptor]['A_delay']
+                    nc_dict['A_weight'] = drive_conn[
+                        receptor]['A_weight']
+                    loc = drive_conn['location']
+                    self._all_to_all_connect(
+                        self.connectivity_list, drive['name'], target_cell,
```

Comment:
```suggestion
                        self.connectivity, drive['name'], target_cell,
```

can we change this consistently.